.circleci/config.yml

```diff
@@ -89,18 +89,23 @@ references:
         echo "8dbad6683d6fc9367e637e6eed8e01a0d63c9660  goreleaser.deb" | sha1sum -c
         sudo dpkg -i goreleaser.deb
         rm goreleaser.deb
+
+  docker_build: &docker_build
+    run:
+      name: Docker login, build, and push
```

Comment:
Could we update this name? Not sure if it has any implications outside of clarity, but I think that's reason enough for a simple change

Reply:
Good catch!